Use correct version of BIO_info_cb (we may need to fix this in libcrypto)

Index: libfreerdp/crypto/tls.c
--- libfreerdp/crypto/tls.c.orig
+++ libfreerdp/crypto/tls.c
@@ -629,7 +629,7 @@ static int bio_rdp_tls_free(BIO* bio)
 	return 1;
 }
 
-static long bio_rdp_tls_callback_ctrl(BIO* bio, int cmd, bio_info_cb* fp)
+static long bio_rdp_tls_callback_ctrl(BIO* bio, int cmd, BIO_info_cb* fp)
 {
 	long status = 0;
 
@@ -648,7 +648,7 @@ static long bio_rdp_tls_callback_ctrl(BIO* bio, int cm
 			typedef void (*fkt_t)(const SSL*, int, int);
 
 			/* Documented since https://www.openssl.org/docs/man1.1.1/man3/BIO_set_callback.html
-			 * the argument is not really of type bio_info_cb* and must be cast
+			 * the argument is not really of type BIO_info_cb* and must be cast
 			 * to the required type */
 
 			fkt_t fkt = WINPR_FUNC_PTR_CAST(fp, fkt_t);
